app/pages/donate.vue

```diff
@@ -1,27 +1,65 @@
 <template>
   <div>
-    <SubTitle title="Lahmacun Donate" :maintitle="true" />
+    <SubTitle title="Lahmacun donation" :maintitle="true" />
     <div class="container my-8">
       <div v-if="$fetchState.pending" class="center">
         Loading...
       </div>
-      <article id="donate-page" ref="donate">
-        <div v-if="donateContent">
-          <h2>{{ donateContent.title.rendered }}</h2>
-          <div v-sanitize="[sanitizeOptions, donateContentResults]" class="donate-content" />
+
+      <div v-if="donationContent" class="max-w-4xl mx-auto">
+        <div class="mb-4">
+          <h2>{{ donationContent.title.rendered }}</h2>
+        </div>
+        <div class="grid md:grid-cols-2 md:gap-8 gap-4">
+          <div>
+            <div v-sanitize="[sanitizeOptions, donationContent.content.rendered]" />
+          </div>
+
+          <div>
+            <form :action="$config.donationStripeFormUrl" method="GET">
+              <div>
+                <p class="mb-2">
+                  <strong>
+                    You can choose from the following options:
+                  </strong>
+                </p>
+
+                <div class="radios">
+                  <v-app>
+                    <v-container>
+                      <v-radio-group v-model="is_recurring">
+                        <v-radio :label="donationContent?.acf?.one_time" value="no" color="black"></v-radio>
+                        <v-radio :label="donationContent?.acf?.recurring" value="yes" color="black"></v-radio>
+                      </v-radio-group>
+                    </v-container>
+                  </v-app>
+                </div>
+                <input type="hidden" name="is_recurring" :value="is_recurring">
+
+              </div>
+              <button type="submit" id="checkout-button">{{ donationContent?.acf?.checkout }}</button>
+            </form>
+            <p>Cancel your subscription
+              <NuxtLink to="/donate-cancel">
+                here
+              </NuxtLink>
+            </p>
+          </div>
+
         </div>
-      </article>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
-import { donateURL } from '~/constants'
```

Comment:
can we remove this `donateURL` from the `'~/constants'`?
or is it used anywhere else?